addons/account_edi/models/account_move.py

```diff
@@ -131,6 +131,19 @@ def _compute_edi_show_abandon_cancel_button(self):
                     move.edi_show_abandon_cancel_button = True
                     break
 
+    def _is_edi_invoice(self):
```

Comment:
Maybe put it as a lambda in get_move_applicability

Reply:
@jco-odoo what should I use as a condition in account_edi_document._postprocess_cancel_edi_results ? Maybe the refactor missed using _get_move_applicability in this case ?

Reply:
Remove method, possibly make separate PR to change condition in postprocess.